components/dashboard/src/Login.tsx

```diff
@@ -123,16 +113,13 @@ export const Login: FC<LoginProps> = ({ onLoggedIn }) => {
 
     return (
         <div id="login-container" className="z-50 flex w-screen h-screen">
-            <div id="login-section" className={"flex-grow flex w-full" + (showWelcome ? " lg:w-1/2" : "")}>
-                <div
-                    id="login-section-column"
-                    className={"flex-grow max-w-2xl flex flex-col h-100 mx-auto" + (showWelcome ? " lg:my-0" : "")}
-                >
+            <div id="login-section" className={"flex-grow flex w-full"}>
+                <div id="login-section-column" className={"flex-grow max-w-2xl flex flex-col h-100 mx-auto"}>
```

Comment:
**thought**: Sound safe to remove, right? Cc @selfcontained because https://github.com/gitpod-io/gitpod/pull/17587.